config/services/civilization-iv.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Sid Meier's Civilization IV</short>
+  <description>
+    Civilization IV is a 4X turn-based strategy computer game and the fourth
+    installment of the Civilization series.
+  </description>
+  <port port="2033" protocol="tcp"/>
+  <port port="2056" protocol="tcp"/>
+  <port port="3783" protocol="tcp"/>
+  <port port="6500" protocol="tcp"/>
+  <port port="6667" protocol="tcp"/>
+  <port port="28900" protocol="tcp"/>
+  <port port="29900-29901" protocol="tcp"/>
+  <port port="2033" protocol="udp"/>
```

Comment:
I think this should be [2056][1], not 2033. Do you agree?
Although, [this forum][2] indicates 2033 and 2056.

[1]: https://portforward.com/civilization-iv/
[2]: https://forums.civfanatics.com/threads/unofficial-civ4-gamespy-multiplayer-firewall-router-port-forwarding-guide.149874/

Reply:
According to [this site](https://www.pcgamingwiki.com/wiki/Sid_Meier%27s_Civilization_IV), neither 2033 nor 2056 udp is required.
With only 2033 it worked the last time I tested it. Unfortunately I currently don't have the possibility to test it without port 2033 opened.